service/provider/src/main/java/gov/va/vro/service/provider/services/EvidenceSummaryDocumentProcessor.java

```diff
@@ -16,11 +18,15 @@ public class EvidenceSummaryDocumentProcessor implements Processor {
 
   @Override
   public void process(Exchange exchange) {
+    String body = exchange.getIn().getBody(String.class);
+    log.info(">>>>> exchange > body: " + body);
```

Comment:
Let's comment these out in case there's any PII in the `body`.
```suggestion
    //log.info(">>>>> exchange > body: " + body);
```

Reply:
Agreed, and done.